snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -160,11 +160,7 @@ open class Snapshot: NSObject {
                 return
             }
             
-            guard let window = app.windows.allElementsBoundByIndex.first(where: { $0.frame.isEmpty == false }) else {
-                print("Couldn't find an element window in XCUIApplication with a non-empty frame.")
-                return
-            }
-
+            let window = window.firstMatch
```

Comment:
I _think_ this line should really be `let window = app.windows.firstMatch`?

Reply:
Yes it is, sorry, copy and paste is so dangerous jejeje. I have it fixed.